ui/app/services/auth.js

```diff
@@ -21,20 +21,25 @@ export { TOKEN_SEPARATOR, TOKEN_PREFIX, ROOT_PREFIX };
 
 export default Service.extend({
   permissions: service(),
+  store: service(),
+  router: service(),
   namespaceService: service('namespace'),
+
   IDLE_TIMEOUT: 3 * 60e3,
   expirationCalcTS: null,
   isRenewing: false,
   mfaErrors: null,
+  activeClusterId: null,
 
-  init() {
-    this._super(...arguments);
-    this.checkForRootToken();
+  get tokenExpired() {
```

Comment:
Can we mix getters and computed properties? Is it worth maybe updating the `computed` properties below to getters instead?

Reply:
I guess to that point some of these computed properties also might not need to be recomputed either. I think I would feel more comfortable tackling this in a separate task to glimmerize the service though in case it does break something.